ecs-cli/modules/cli/compose/project/project_parseV1V2.go

```diff
@@ -0,0 +1,14 @@
+package project
+
+import (
+	"github.com/docker/libcompose/project"
+	"github.com/sirupsen/logrus"
+)
+
+func (p *ecsProject) parseV1V2() {
+	libProject := project.NewProject(&p.ecsContext.Context, nil, nil)
+	libProject.Parse()
+
+	logrus.Debug("Parsing v1/2 project...")
```

Comment:
Total nit, but since we have this debug log statement in both `parseV1V2` and `parseV3`, would it make sense to move it into the `parseCompose` function itself and have something like `logrus.Debugf("Parsing docker compose version: %v", version)`?